### PR TITLE
fix issue #4046, update toolbar documentation

### PR DESCRIPTION
**Pull request by mrosati84**

### CODE REVIEW

**Comment by evildmp** [comment]
Taking a look now.

File: docs/upgrade/3.1.rst [hunk 1]
```diff
@@ -152,6 +152,9 @@ Improvements to the toolbar user interface
 ==========================================
 
 Some minor changes have been implemented to improve the toolbar user interface.
+The old *Draft/Live* switch has been replaced to achieve a more clear
```
Comment (evildmp) [inline docs/upgrade/3.1.rst]: Note that buttons that perform an action such **Draft/Live** as this should be marked up with strong emphasis: http://docs.django-cms.org/en/develop/contributing/contributing.html#inline-markup (don't worry, I'll make these changes here).

**Comment by evildmp** [comment]
Thanks Matteo, this code has now been merged in https://github.com/divio/django-cms/pull/4057.

**Comment by yakky** [comment]
Fixed in #4057

**Pull request closed without merging**